mcs/class/corlib/Test/System/TypeTest.cs

```diff
@@ -435,6 +443,16 @@ public void TestIsAssignableFrom ()
 			Assert.IsFalse (typeof (Array).IsAssignableFrom (gparams[0]), "#45");
 			Assert.IsFalse (typeof (Delegate).IsAssignableFrom (gparams[0]), "#46");
 
+			// Arrays of generic parameters and arrays of interfaces, see #15749
```

Comment:
nit: we've had a bunch of bug trackers for Mono over the years.  If you get a chance to update the PR one more time, put in a full URL.